pkg/asset/installconfig/openstack/validation/platform.go

```diff
@@ -124,6 +124,9 @@ func validateFloatingIPs(p *openstack.Platform, ci *CloudInfo, fldPath *field.Pa
 		} else if p.ExternalNetwork == "" {
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("ingressFloatingIP"), p.IngressFloatingIP, "Cannot set floating ips when external network not specified"))
 		}
+		if p.APIFloatingIP != "" && p.APIFloatingIP == p.IngressFloatingIP {
```

Comment:
`APIFloatingIP` doesn't exist in release-4.6, it's only in 4.7 that we renamed LbFloatingIP to APIFloatingIP with https://github.com/openshift/installer/pull/4244/.

Given the low severity of the bz, I wonder if it's really worth backporting this patch.